src/ResidentDirectory.js

```diff
@@ -97,11 +100,28 @@ class ResidentDirectory extends Component {
                     .then((text) => {
                         alert(text.error);
                     });
+            } else if (response.status === 200) {
+                this.setState({
+                    displayCard: false,
+                    value: '?',
+                    data: '',
```

Comment:
- Should have a default value in constructor for clarity. 
- Naming is obscure.